### ISSUE TITLE
Inline stem not quoted in the latex backend

### ISSUE THREAD

**Issue by ProgramFan** [asciidoctor/asciidoctor-latex#75]
https://github.com/asciidoctor/asciidoctor-latex/blob/652176e5f94ad7972a0d0bb4cc6721ce7a35fe93/lib/asciidoctor/latex/node_processors.rb#L801

Caused directly by the above line.

**Comment by KaminariOS** [comment]
Yeah, I think [latexmath]: inline syntax should be wrapped so that we can write 100%  compatible Asciidoc.
